webpack.config.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.ts',
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: 'ts-loader',
+        exclude: /node_modules/
+      }
+    ]
+  },
+
+  // Enable sourcemaps for debugging webpack's output.
+  devtool: "source-map",
+
+  resolve: {
+    extensions: [ '.tsx', '.ts', '.js' ]
+  },
+  output: {
+    filename: 'app.js',
+    path: path.resolve(__dirname, 'public')
+  },
+  externals : {
+    microsoftTeams: '@microsoft/teams-js'
```

Comment:
Why is this needed? Can't you just import microsoftTeams wherever it's needed?

Reply:
to make it not be part of final outPut JS as i am planning to reference teams js separately